Packs/DigitalShadows/ReleaseNotes/2_0_1.md

```diff
@@ -0,0 +1,11 @@
+
+#### Integrations
+
+##### New: ReliaQuest GreyMatter DRP Event Collector
```

Comment:
I saw an integration and not a data collector - is that intentional?

Reply:
no it missed it must be "ReliaQuest GreyMatter DRP Incidents"